docs/backend/relations.md

```diff
@@ -0,0 +1,622 @@
+        value_type=RelationChoice(
+            source=CatalogSource(
+                portal_type=["one_eyed_minion", "minion"],
+                review_state="published",
+            )
+        ),
+        required=False,
+    )
+    directives.widget(
+        "minions",
+        RelatedItemsFieldWidget,
+        pattern_options={
+            "mode": "auto",
+            "favorites": minion_favorites,
+            }
+        )
+```
+
+
+(relations-relationfields-through-the-web-or-in-xml-label)=
+
+## RelationFields through the web or in xml
+
+It is surprisingly easy to create RelationFields through the web.
+
+- Using the Dexterity schema editor, add a new field and select _Relation List_ or _Relation Choice_, depending on whether you want to relate to multiple items or not.
+- When configuring the field you can even select the content type the relation should be limited to.
+
+When you click on `Edit XML field model` you will see the fields in the XML schema:
+
+RelationChoice:
+
+```{code-block} python
+:linenos:
+
+<field name="boss" type="z3c.relationfield.schema.RelationChoice">
+  <description/>
+  <required>False</required>
+  <title>Boss</title>
+</field>
+```
+
+RelationList:
+
+```{code-block} python
+:linenos:
+
+<field name="underlings" type="z3c.relationfield.schema.RelationList">
+  <description/>
+  <required>False</required>
+  <title>Underlings</title>
+  <value_type type="z3c.relationfield.schema.RelationChoice">
+    <title i18n:translate="">Relation Choice</title>
+    <portal_type>
+      <element>Document</element>
+      <element>News Item</element>
+    </portal_type>
+  </value_type>
+</field>
+```
+
+
+(relations-using-different-widgets-for-relations-label)=
+
+## Using different widgets for relations
+
+```{todo}
+Support for `StaticCatalogVocabulary` in Volto is currently still missing.
+```
+
+Often the standard widget for relations is not what you want since it can be hard to navigate to the content you want to relate to and the search-mode of the default widget is not suitable for all use-cases.
+
+If you want to use checkboxes, radiobuttons or a selection-dropdown you need to use `StaticCatalogVocabulary` instead of `CatalogSource` to specify your options.
+
+```{code-block} python
+:emphasize-lines: 8, 18
+:linenos:
+
+from plone.app.vocabularies.catalog import StaticCatalogVocabulary
+from plone.app.z3cform.widget import SelectFieldWidget
+from plone.autoform import directives
+from z3c.relationfield.schema import RelationChoice
+
+relationchoice_field_select = RelationChoice(
+    title="RelationChoice with Select Widget",
+    vocabulary=StaticCatalogVocabulary(
+        {"portal_type": ["Document", "Event"]}
+    ),
+    required=False,
+)
+directives.widget(
+    "relationchoice_field_select",
+    SelectFieldWidget,
+)
+```
+
+The field should then look like this:
+
+```{figure} /_static/relation_select.png
+:alt: RelationList field with select widget
+
+RelationList field with select widget
+```
```

Comment:
The above alt text and caption do not align with the code or the text in the image, "RelationChoice with Select Widget". Is this correct?

Reply:
Fixed it